lib/rubocop/config_obsoletion/changed_enforced_styles.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RuboCop
+  class ConfigObsoletion
+    # Encapsulation of a ConfigObsoletion rule for changing a parameter
+    # @api private
+    class ChangedEnforcedStyles < ParameterRule
+      def violated?
+        style = config[cop]&.detect { |key, _| key.start_with?(parameter) }
```

Comment:
What are `key` and `parameter` here? Why can't we `==` them?

Reply:
I've changed it to just be a straight up `==`, we can extend it in the future if something comes up to make it necessary.